src/consensus/params.h

```diff
@@ -7,8 +7,29 @@
 #define BITCOIN_CONSENSUS_PARAMS_H
 
 #include "uint256.h"
+#include <map>
+#include <string>
 
 namespace Consensus {
+
+enum DeploymentPos
```

Comment:
Adding a comment for doxygen would also help if the purpose is not immediately clear from the code alone.